src/app/personal-details/page.tsx

```diff
@@ -37,225 +38,175 @@ const PersonalDetailsScreen: React.FC = () => {
 			}}
 		>
 			<Container
-				maxWidth="md"
+				maxWidth="sm"
 				sx={{
 					backgroundColor: "#FFFFFF",
 					padding: "48px",
 					borderRadius: "16px",
 					border: "1px solid #D6D6E7",
 					boxShadow: "0px 4px 12px rgba(0, 0, 0, 0.1)",
+					textAlign: "center",
 				}}
 			>
 				{/* Header */}
 				<Typography
 					variant="h5"
 					component="h1"
 					sx={{
-						textAlign: "center",
 						color: "#000000",
 						fontWeight: 600,
 						fontSize: "32px",
 						lineHeight: "40.32px",
-						marginBottom: "24px",
+						marginBottom: "32px",
 						fontFamily: "'Outfit', sans-serif",
 					}}
 				>
-					Tell Us About Yourself
+					My Profile
 				</Typography>
 				<Typography
 					variant="body2"
 					sx={{
-						textAlign: "center",
-						color: "#000000",
-						marginBottom: "32px",
-						fontSize: "24px",
-						lineHeight: "30.24px",
+						color: "#6C6C80",
+						marginBottom: "46px",
+						fontSize: "20px",
+						lineHeight: "28px",
 						fontFamily: "'Outfit', sans-serif",
-						fontWeight: 200,
 					}}
 				>
-					This helps other founders understand who you are when reviewing your
-					ideas.
+					This helps others understand who you are when reviewing your ideas.
```

Comment:
Love how you're using general language that applies to both before & after the Minerva capstone / founders pivots! Great adjustment here -- thank you!